app/views/overview/_list-row-actions.html

```diff
@@ -28,6 +28,9 @@
             <li ng-if="('pod_presets' | enableTechPreviewFeature) && row.state.bindableServiceInstances.length" role="menuitem">
               <a href="" ng-click="row.showOverlayPanel('bindService', {target: row.apiObject})">Create Binding</a>
             </li>
+            <li ng-if="('pod_presets' | enableTechPreviewFeature) && row.state.bindableServiceInstances.length" role="menuitem">
```

Comment:
Maybe use `ng-if-start` on the `li` above and `ng-if-end` since the conditions are the same?